chainer/backends/cuda.py

```diff
@@ -121,7 +122,7 @@ class _PerformanceWarning(Warning):
 
 
 if available:
-    _cupy_major = numpy.lib.NumpyVersion(cupy.__version__).major
+    _cupy_major = packaging.version.Version(cupy.__version__).major
```

Comment:
Let's avoid introducing non-standard library dependency :smile:
```suggestion
    _cupy_major = int(cupy.__version__.split('.')[0])
```